### PR TITLE
Sc22

### PR DESCRIPTION
**Pull request by jyothibv**
*Issue #, if available:*

*Description of changes:*
Version 1 of Lab 2 ( Lustre + S3 DRA ) for SC22. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/08-fsx-lustre/02-creates3-upload.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Navigate to your Cloud9 IDE and on the Cloud9 terminal create S3 bucket with unique name.
+
+{{% notice info %}}
+The bucket name must start with **s3://**.
+Choose a random prefix, postfix, or append your name. 
+{{% /notice %}}
+
+```bash
+BUCKET_POSTFIX=$(uuidgen --random | cut -d'-' -f1)
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/02-creates3-upload.md:20]:
```suggestion
BUCKET_POSTFIX=$(python3 -S -c "import uuid; print(str(uuid.uuid4().hex)[:10])")
BUCKET_NAME_DATA="bucketname-${BUCKET_POSTFIX}"
```

File: content/08-fsx-lustre/02-creates3-upload.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+aws s3 mb s3://bucket-${BUCKET_POSTFIX}
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/02-creates3-upload.md:21]:
```suggestion
aws s3 mb s3://${BUCKET_NAME_DATA}
```

**Review by mhuguesaws — COMMENTED** [review]
Left comments.

File: content/08-fsx-lustre/01-create-fsxl.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+![createfsl](/images/fsx-for-lustre-hsm/createfsl.png)
+
+10. Creating this file system is expected to take around 10 mins. In the meantime let us move to next steps. 
+
+11. While the file system is creating, you will need to edit the inbound rules of the security group attached to this file system to allow EC2 instances to mount the file system. For this, go on the AWS console, search for VPC and click VPC service. On this page, on the left most column you will find a link to **security groups**. Click on this link and look for the security group ( compute cluster security group ) you noted in step 6. In the details of this security group click on **Edit inbound rules** button.
+
+![editinbound](/images/fsx-for-lustre-hsm/editinbound.png)
+
+12. This will open a screen where you need to click on **Add rule** button and add **Custom TCP** , port number **988** , from 0.0.0.0 ( **Note** this is not a very good practice but this is in the interest of time for this lab. However a more restricted security group of the instance to be connected is a more advisible source rather than allowing it everywhere). Once this is done, click on **Save rules**.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/01-create-fsxl.md:64]: Not sure that is needed.

**Review by mhuguesaws — COMMENTED** [review]
left comments.

File: content/08-fsx-lustre/01-create-fsxl.md (hunk 3 above)
Comment (jyothibv) [reply content/08-fsx-lustre/01-create-fsxl.md:64]: I tested this without the port open and I could not mount. With this rule added mount succeeds

File: content/08-fsx-lustre/01-create-fsxl.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+title = "a. Create FSx for Lustre fielsystem via FSx Console"
+date = 2019-09-18T10:46:30-04:00
+weight = 80
+tags = ["tutorial", "lustre", "FSx"]
++++
+
+In this section, you will create a new FSx for lustre filesystem in your test account
+
+
+1. On your AWS console main page search for FSx and select and click on FSx service from the drop down.
```
[the remark this answers is not in the document]
Comment (jyothibv) [reply content/08-fsx-lustre/01-create-fsxl.md:11]: Will do. Thanks

**Comment by sean-smith** [comment]
Overall, wondering why we're creating a filesystem **external** to AWS ParallelCluster. This seems overkill when we can just create a FSxL P2 filesystem with the cluster.

File: content/08-fsx-lustre/01-create-fsxl.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jyothibv) [reply content/08-fsx-lustre/01-create-fsxl.md:11]: Addressed.Updated pull request.

File: content/08-fsx-lustre/02-creates3-upload.md (hunk 2 above)
Comment (jyothibv) [reply content/08-fsx-lustre/02-creates3-upload.md:21]: Addressed, updated pull request

File: content/08-fsx-lustre/02-creates3-upload.md (hunk 1 above)
Comment (jyothibv) [reply content/08-fsx-lustre/02-creates3-upload.md:20]: Addressed, updated pull request

**Pull request merged**